### PR TITLE
feat(route): ktown4u

### PR DESCRIPTION
**Pull request by JamesWDGu**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/ktown4u/artistBrandlist/234590
/ktown4u/artistBrandlist/234590/1723449
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明
没有时区信息，国区网站，默认中国时间

### CODE REVIEW

File: lib/routes/ktown4u/artist-brandlist.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['cn.ktown4u.com/artistBrandlist?grp_no=:grp_no&grp_no2=:grp_no2'],
```
Comment (TonyRL) [inline lib/routes/ktown4u/artist-brandlist.ts:20]: `source` can't match any search parameters.

File: lib/routes/ktown4u/artist-brandlist.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        const items = data.map((item) => ({
+            title: item.GOODS_NM,
+            url: item.IMG_PATH,
+            link: `https://cn.ktown4u.com/iteminfo?goods_no=${item.GOODS_NO}`,
+            description: desc(item),
+            pubDate: parseDate(item.RELEASE_DT),
+        }));
+
+        return {
+            title: `${String(rssTitle(data))}`,
```
Comment (TonyRL) [inline lib/routes/ktown4u/artist-brandlist.ts:45]: `rssTitle` returns a `string`. Using `String()` on a string is unnecessary.

File: lib/routes/ktown4u/artist-brandlist.ts (hunk 1 above)
Comment (JamesWDGu) [reply lib/routes/ktown4u/artist-brandlist.ts:20]: Thanks @TonyRL. Actually, I'm not sure what this does, how would you recommend I modify it, or can I delete it?

File: lib/routes/ktown4u/artist-brandlist.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    radar: [
+        {
+            source: ['cn.ktown4u.com/artistBrandlist?grp_no=:grp_no&grp_no2=:grp_no2'],
+            target: '/artistBrandlist/:grp_no/:grp_no2',
+        },
+    ],
+    name: 'Get the products on sale',
+    maintainers: ['JamesWDGu'],
+    handler: async (ctx) => {
+        const { grp_no, grp_no2 = '' } = ctx.req.param();
```
Comment (TonyRL) [inline lib/routes/ktown4u/artist-brandlist.ts:27]: `camelCase`

File: lib/routes/ktown4u/artist-brandlist.ts (hunk 2 above)
Comment (JamesWDGu) [reply lib/routes/ktown4u/artist-brandlist.ts:45]: it's a lint suggestion
>  45:20  error  Unexpected implicit coercion encountered. Use `String(rssTitle(data))` instead  no-implicit-coercion

File: lib/routes/ktown4u/artist-brandlist.ts (hunk 1 above)
Comment (TonyRL) [reply lib/routes/ktown4u/artist-brandlist.ts:20]: delete it

Comment (JamesWDGu) [reply lib/routes/ktown4u/artist-brandlist.ts]: deleted

File: lib/routes/ktown4u/artist-brandlist.ts (hunk 2 above)
Comment (TonyRL) [reply lib/routes/ktown4u/artist-brandlist.ts:45]: I guess you're trying to do
```ts
title: `${rssTitle(data)}`
```
which has unnecessary interpolations. You can simply do
```ts
title: rssTitle(data)
```

Comment (JamesWDGu) [reply lib/routes/ktown4u/artist-brandlist.ts:45]: Got it, thanks!

Comment (JamesWDGu) [reply lib/routes/ktown4u/artist-brandlist.ts]: fixed it

File: lib/routes/ktown4u/artist-brandlist.ts [hunk 4]
```diff
@@ -0,0 +1,61 @@
+import { Route } from '@/types';
+import ofetch from '@/utils/ofetch';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/artistBrandlist/:grpNo/:grpNo2?',
+    categories: ['shopping'],
+    example: 'ktown4u/artistBrandlist/234590/1723449',
```
Comment (TonyRL) [inline lib/routes/ktown4u/artist-brandlist.ts:8]:
```suggestion
    example: '/ktown4u/artistBrandlist/234590/1723449',
```

**Comment by JamesWDGu** [comment]
Hi @TonyRL, I was wondering if we merged it, when can I access it via follow's rsshub service

**Comment by TonyRL** [comment]
Building the docker image takes [roughly 15 mins](https://github.com/DIYgod/RSSHub/actions/workflows/docker-release.yml). It also takes some time for the cluster to roll the new image. I would suggest wait for at least an hour.

**Pull request merged**